packages/babel-plugin-proposal-decorators/src/index.js

```diff
@@ -172,10 +172,18 @@ export default function() {
                 decorators.map(dec => t.cloneNode(dec.expression)),
               ),
               t.objectExpression([
+                t.objectProperty(
```

Comment:
i'd suspect this change touch some `output.js` test file, but this PR has no such changes, do you know why?

Reply:
Sorry for the late reply. There are only 4 fixture-based tests in this package and one of them is affected by this change. I've already included the change in this commit (`test/fixtures/decl-to-expression/method-decorators/output.js`). Since there's no plugin depending on this package, no tests outside this package would be affected by this change.